book/urban/sensors/urban-sensors-ukv.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<p>It would be helpful to add a legend to facilitate the reader the understanding of the plot. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/acocac/environmental-ai-book/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done!</p>